services/ui-src/src/utils/testing/mockUsers.tsx

```diff
@@ -20,6 +20,25 @@ export const mockStateUser: UserContextShape = {
     full_name: "Thelonious States",
     state: "MN",
     userIsEndUser: true,
+    userReports: ["MCPAR", "MLR", "NAAAR"],
+  },
+  showLocalLogins: true,
+  logout: async () => {},
+  loginWithIDM: () => {},
+  updateTimeout: async () => {},
+  getExpiration: () => {},
+};
+
+export const mockStateUserNoReports: UserContextShape = {
+  user: {
+    userRole: UserRoles.STATE_USER,
+    email: "stateuser@test.com",
+    given_name: "Thelonious",
+    family_name: "States",
+    full_name: "Thelonious States",
+    state: "MN",
+    userIsEndUser: true,
+    userReports: undefined,
```

Comment:
Question: Do we need to also test for if a user has selected no reports when they provision their role? Idk if state users will have undefined (all other user types definitely will though), but i think they might have the IDM reports attribute equal to an empty string `""` if they didn't select any reports. We should check on this/test for it I think.

Reply:
good call, I changed this unit test to have an array of an empty string, just for consistency across state user roles.